tests/detect-flowbits/test.rules

```diff
@@ -0,0 +1,12 @@
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,fb1; sid:1;)
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,fb2; sid:2;)
+alert ip any any -> any any (msg:"Flowbit isset ored flowbits"; flowbits:isset,fb3|fb4; sid:3;)
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,fb5; sid:5;)
+alert ip any any -> any any (msg:"Flowbit isset ored flowbits"; flowbits:isset,fb5|fb6; sid:6;)
+alert ip any any -> any any (msg:"Flowbit isnotset ored flowbits"; flowbits:isnotset, fb1 | fb2 ; sid:7;)
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,myflow; sid:8;)
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,myflow2; sid:9;)
+alert ip any any -> any any (msg:"Flowbit unset"; flowbits:unset,myflow2; sid:10;)
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:noalert; flowbits:toggle,myflow2; content:"GET "; sid:11;)
+alert ip any any -> any any (msg:"Noalert"; flowbits:noalert; content:"GET "; sid:12;)
+alert ip any any -> any any (msg:"isset option"; flowbits:isset,fbt; content:"GET "; sid:13;)
```

Comment:
I do not see any tests for `AND` op among flowbits. Please add those rules too. e.g. `isset,fb1,fb2,..`
You can read more here: https://unixia.wordpress.com/2020/05/17/more-checks-with-lesser-rules-a-flowbit-magic-tale/